tests/operator/operator.go

```diff
@@ -2576,6 +2604,11 @@ spec:
 				managedByChanged    = "someone"
 			)
 
+			BeforeEach(func() {
```

Comment:
I wonder if we can simply not play with the feature toggle and see if adoption is working.

Reply:
Also can we merge these test?

Reply:
Testing adoption requires the target object to exist before the operator reconciles so I think we need to start with the feature disabled here. I can however merge the tests into one.